### PR TITLE
chore(remove-script-prefix): package.json - Remove --prefix

### PR DESCRIPTION
**Pull request by Zainzzkk**
## Introduction :pencil2:

This PR removes --prefix from package.json and replaces it with cd commands

## Resolution :heavy_check_mark:

* Removed --prefix from package.json scripts
* Added cd to directories instead
* Fixed linting error in api directory

### CODE REVIEW

File: src/api/types/test-helpers/index.ts
Comment (abhi-markan) [inline src/api/types/test-helpers/index.ts:2]: Can you please add a comment why we are done this and a `TODO` on a long term fix on the ticket please.

**Review by abhi-markan — CHANGES_REQUESTED** [review]
One minor comment please.

File: src/api/types/test-helpers/index.ts
Comment (Zainzzkk) [reply src/api/types/test-helpers/index.ts:6]: done - no TODO as not sure if it will ever be fixed

**Review by ttbarnes — CHANGES_REQUESTED** [review]
Can we please hold off of merging this until we have `no-pdf` merged into `main`.

Otherwise will have to merge `no-pdf` into `main` again. Will be quicker to have this branch/PR rebased, or apply the changes in a new branch off of `main`

**Review by ttbarnes — APPROVED** [review]

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**
